invenio_rdm_records/records/jsonschemas/records/definitions-v1.2.0.json

```diff
@@ -27,8 +27,15 @@
       }
     ]
   },
-  "contributorType": {
-    "type": "string"
+  "roles": {
```

Comment:
Shouldn't it be `role` since it is only one? then the contirbutor has a `role` with each element being a role ref?